modules/person.rb

```diff
@@ -0,0 +1,63 @@
+require './nameable'
```

Comment:
- [ ] Kindly change this to `require_relative` as that is the convention for importing local files in ruby. I was unable to run your app due to an error from this. The `require` is reserved for external files like gems.